docs/topics/cli-run.adoc

```diff
@@ -38,7 +38,9 @@ The following command analyzes the `com.acme` and `org.apache` packages of the l
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/{mta-cli} \
+
+$ <{ProductShortName}_HOME>/bin/windup-cli \
+
     --input /path/to/jee-example-app-1.0.0.ear \
     --output /path/to/report-output/ --source eap:5 --target eap:7 \
     --packages com.acme org.apache
```

Comment:
The `--packages` argument no longer exists

Reply:
@abrugaro - so this is where the confusion comes into play

**cli-run.adoc** is for  MTR. If we change this file for MTA specific it will change MTR docs.

So what I have done is rollback **cli-un.adoc** and created **mta-cli-run.adoc** to hold the changes for MTA

Making this change, will help when I have to move MTA docs out of the WINDUP repo

It was a mistake to include MTA nad MTR in the same doc and calling the same files as they would obviously likely diverge and cause these problems.

WHat do you think?

Reply:
Changes split between `cli-run.adoc` and `mta-cli-run.adoc`

```
/home/anarnold/Documents/SublimeMerge/windup-documentation/docs/cli-guide-mtr/master.adoc:
   32  
   33  // Run the CLI
   34: include::topics/cli-run.adoc[leveloffset=+2]
   35  
   36  // Use OpenRewrite recipes

windup-documentation/docs/cli-guide/master.adoc:
   36  
   37  // Run the CLI
   38: include::topics/mta-cli-run.adoc[leveloffset=+2]
```